ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -60,8 +60,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-c2s-ipi-disc-priv-fips-f28
-  cron: 45 15 2 * *
+- as: aws-c2s-ipi-disc-priv-fips-f60
```

Comment:
This is Y-stream upgrade ci jobs, not z-stream upgrade ci, right?

Reply:
faint...time always wasted by such stupid issue